docs/using-kontena/loadbalancer.md

```diff
@@ -231,7 +231,7 @@ These options are defined on the services that are balanced through Kontena Load
 * `KONTENA_LB_INTERNAL_PORT`: service port that is attached to load balancer
 * `KONTENA_LB_EXTERNAL_PORT`: service port that load balancer starts to listen (only for tcp mode)
 * `KONTENA_LB_VIRTUAL_HOSTS`: comma-separated list of virtual hosts, if you would like to access your service through the vpn, you'll need to add that as a virtual host as well (only for http mode)
-* `KONTENA_LB_VIRTUAL_PATH`: path that is used to match request; example: "/api" (only for http mode)
-* `KONTENA_LB_KEEP_VIRTUAL_PATH`: if set to true, virtual path will be kept in request path (only for http mode)
+* `KONTENA_LB_VIRTUAL_PATH`: path that is used to match request; example: "/api" (only for http mode). By default, a request to e.g. `loadbalancer/api/foo` is proxied to e.g. `service/foo`
```

Comment:
Yes, but now `loadbalancer` and `service` gets now easily confused as being part of the path. Maybe something like:
By default the path is omitted from the forwarded request, so a request to path `/api/foo` is proxied with path `/foo`